<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Leica</class>
 <widget class="QMainWindow" name="Leica">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Leica</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbBaud">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Baud rate</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbBaud">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>76</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbPort">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbPort">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>76</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbOp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>271</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir porta serial</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbStatus">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Aguardando usu√°rio...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
